### PR TITLE
Fix/2856/action icon

### LINKED ISSUE #2856: Unify UI of action icons in toolbar
## Expected Behavior
As a user I want a consistent UI, as every deviation takes away focus from my task.

## Actual Behavior
The icon of "Download stl file for 3D printing" is not aligned vertically.

It and the global configuration button to the right have no hover effect.

## Screenshots
image

## Open questions
I would like to replace the icon of the  "Download stl file for 3D printing"-button, as it isn't clear on first glance that it is a download and it is the only one of the buttons which uses mat-icon instead of an fa icon. What do you think about [fa-arrow-circle-o-down](https://fontawesome.com/v4/icon/arrow-circle-o-down) or [cube](https://fontawesome.com/v4/icon/cube)

## Specifications
- is released in [online-demo](https://maibornwolff.github.io/codecharta/visualization/app/index.html?file=codecharta.cc.json&file=codecharta_analysis.cc.json): yes | no
- CodeCharta Version: 1.98.0

## Development notes (optional Task Breakdown)
- [ ]  Introduce `cc-action-icon` component, which unifies the styles first

### PR DESCRIPTION
**Pull request by shaman-apprentice**
close #2856, ref #2318

### CODE REVIEW

**Comment by shaman-apprentice** [comment]
Changelog is still missing. Will add depending on chosen Icon

File: visualization/app/codeCharta/ui/actionIcon/actionIcon.component.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	align-items: center;
+	cursor: pointer;
+	width: 24px;
+	height: 24px;
+	border-radius: 100%;
+	background-color: theme.$cc-primary-color;
+	font-size: 16px;
+
+	&:hover {
+		background-color: rgb(0, 124, 112);
```
Comment (IhsenBouallegue) [inline visualization/app/codeCharta/ui/actionIcon/actionIcon.component.scss:14]: Can we extract this to a variable under theme?

File: visualization/app/codeCharta/ui/actionIcon/actionIcon.component.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+cc-action-icon {
+	display: inline-flex;
+	align-items: center;
+	cursor: pointer;
+	width: 24px;
+	height: 24px;
+	border-radius: 100%;
+	background-color: theme.$cc-primary-color;
+	font-size: 16px;
```
Comment (IhsenBouallegue) [inline visualization/app/codeCharta/ui/actionIcon/actionIcon.component.scss:11]:
```suggestion
	font-size: 14px;
```
looks a bit better when we leave some space

File: visualization/app/codeCharta/ui/toolBar/globalConfigurationButton/globalConfigurationButton.component.scss [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
 	margin: 0;
 
-	i {
-		background-color: theme.$cc-primary-color;
-		color: white;
-		border-radius: 100%;
-		padding: 5px;
-		font-size: 14px;
+	&:hover cc-action-icon {
+		background-color: rgb(0, 124, 112);
```
Comment (IhsenBouallegue) [inline visualization/app/codeCharta/ui/toolBar/globalConfigurationButton/globalConfigurationButton.component.scss:7]: Change this to a variable too 👍

**Review by IhsenBouallegue — APPROVED** [review]
Thank you! LGTM🥇

**Review by jannikr — APPROVED** [review]
Looks good on the Mac too!

File: visualization/app/codeCharta/ui/toolBar/globalConfigurationButton/globalConfigurationButton.component.scss (hunk 3 above)
Comment (shaman-apprentice) [reply visualization/app/codeCharta/ui/toolBar/globalConfigurationButton/globalConfigurationButton.component.scss:7]: done with next push

File: visualization/app/codeCharta/ui/actionIcon/actionIcon.component.scss (hunk 2 above)
Comment (shaman-apprentice) [reply visualization/app/codeCharta/ui/actionIcon/actionIcon.component.scss:11]: done with next push

**Pull request merged**